osu.Game.Tests/NonVisual/FirstAvailableHitWindowTest.cs

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using NUnit.Framework;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Rulesets.Judgements;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Osu;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Rulesets.UI;
+using osu.Game.Scoring;
+
+namespace osu.Game.Tests.NonVisual
+{
+    public class FirstAvailableHitWindowsTest
+    {
+        private TestDrawableRuleset testDrawableRuleset = new TestDrawableRuleset();
+
+        [Test]
+        public void TestResultIfOnlyParentHitWindowIsEmpty()
+        {
+            var testObject = new TestHitObject(hitWindows: HitWindows.Empty);
```

Comment:
Not sure why the named parameter part is here; surely the name is implied if the constructor has only the one parameter?